### PR TITLE
docs: refactor loggers plugin doc（1）

### PR DESCRIPTION
**Pull request by hf400159**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/clickhouse-logger.md [hunk 1]
```diff
@@ -23,72 +29,35 @@ title: clickhouse-logger
...
-| database         | string  | 必须   |               |         | 使用的数据库。                                      |
-| logtable         | string  | 必须   |               |         | 写入的表名 。                                       |
-| user             | string  | 必须   |               |         | clickhouse 的用户。                                |
-| password         | string  | 必须   |               |         | clickhouse 的密码 。                               |
-| timeout          | integer | 可选   | 3             | [1,...] | 发送请求后保持连接活动的时间。                       |
-| name             | string  | 可选   | "clickhouse logger" |         | 标识 logger 的唯一标识符。                    |
-| ssl_verify       | boolean | 可选   | true          | [true,false] | 验证证书。                                    |
+| 名称             | 类型    | 必选项  | 默认值              | 有效值       | 描述                                                     |
+| ---------------- | ------- | ------ | ------------------- | ----------- | -------------------------------------------------------- |
+| endpoint_addr    | 废弃    | 是     |                     |              | 请使用 `endpoint_addrs` 代替。ClickHouse 的 `endpoints`。 |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/clickhouse-logger.md:38]:
```suggestion
| endpoint_addr    | 废弃    | 是     |                     |              | 请使用 `endpoint_addrs` 代替 ClickHouse 的 `endpoints`。 |
```

File: docs/zh/latest/plugins/clickhouse-logger.md [hunk 2]
```diff
@@ -23,72 +29,35 @@ title: clickhouse-logger
...
-
-```shell
-$ curl -i http://127.0.0.1:9080/hello
-HTTP/1.1 200 OK
-...
-hello, world
-```
-
-## 插件元数据设置
+`skywalking-logger` 也支持自定义日志格式，与 [http-logger](./http-logger.md) 插件类似。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/clickhouse-logger.md:52]: clickhouse-logger?

File: docs/zh/latest/plugins/clickhouse-logger.md [hunk 3]
```diff
@@ -111,20 +80,61 @@ CREATE TABLE default.test (
...
 │ 127.0.0.1 │ 127.0.0.1 │ 1        │ 2022-01-17T10:03:10+08:00 │
 └───────────┴───────────┴──────────┴───────────────────────────┘
 ```
 
+## 启用插件
+
+你可以通过以下命令在指定路由中启用该插件：
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/clickhouse-logger.md:96]:
```suggestion
curl http://127.0.0.1:9080/apisix/admin/routes/1  \
-H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```

File: docs/zh/latest/plugins/error-log-logger.md [hunk 4]
```diff
@@ -114,16 +105,15 @@ curl http://127.0.0.1:9080/apisix/admin/plugin_metadata/error-log-logger -H 'X-A
 }'
 ```
 
-## 如何设置接收日志的 clickhouse 数据库
+### 如何设置接收日志的 ClickHouse 数据库
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/error-log-logger.md:108]:
```suggestion
### 配置 ClickHouse 数据库
```

File: docs/zh/latest/plugins/error-log-logger.md [hunk 5]
```diff
@@ -134,3 +124,16 @@ curl http://127.0.0.1:9080/apisix/admin/plugin_metadata/error-log-logger -H 'X-A
   }
 }'
 ```
+
+## 禁用插件
+
+当你不在需要该插件是，只需要在 `./conf/config.yaml` 中删除或注释该插件即可。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/error-log-logger.md:130]:
```suggestion
当你不再需要该插件时，只需要在 `./conf/config.yaml` 中删除或注释该插件即可。
```

File: docs/zh/latest/plugins/log-rotate.md [hunk 6]
```diff
@@ -1,5 +1,11 @@
 ---
 title: log-rotate
+keywords:
+  - APISIX
+  - API 网关
+  - Plugin
+  - 日志切分
+description: 云原生 API 网关 Apache APISIX log-rotate 插件用于定期在日志目录下的访问日志和错误日志的定期切分。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/log-rotate.md:8]:
```suggestion
description: 云原生 API 网关 Apache APISIX log-rotate 插件用于将日志目录下的访问日志和错误日志进行定期切分。
```

File: docs/zh/latest/plugins/log-rotate.md [hunk 7]
```diff
@@ -96,6 +95,17 @@ plugin_attr:
         enable_compression: false    # enable log file compression(gzip) or not, default false
 ```
 
-#### 禁用插件
+配置完成，你需要重新加载 APISIX。
 
-在 `conf/config.yaml` 中删除插件 `log-rotate` 即可。
+## 禁用插件
+
+当你不在需要该插件是，只需要在 `./conf/config.yaml` 中删除或注释该插件即可。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/log-rotate.md:102]:
```suggestion
当你不再需要该插件时，只需要在 `./conf/config.yaml` 中删除或注释该插件即可。
```

File: docs/zh/latest/plugins/udp-logger.md [hunk 8]
```diff
@@ -1,5 +1,11 @@
 ---
 title: udp-logger
+keywords:
+  - APISIX
+  - API 网关
+  - Plugin
+  - UDP Logger
+description: 本文介绍了 API 网关 Apache APISIX 如何使用 udp-logger 插件将日志数据发送到 TCP 服务器。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/udp-logger.md:8]:
```suggestion
description: 本文介绍了 API 网关 Apache APISIX 如何使用 udp-logger 插件将日志数据发送到 UDP 服务器。
```

File: docs/zh/latest/plugins/clickhouse-logger.md [hunk 9]
```diff
@@ -23,72 +29,35 @@ title: clickhouse-logger
...
-
-## 插件元数据设置
+`clickhouse-logger` 也支持自定义日志格式，与 [http-logger](./http-logger.md) 插件类似。
 
 | 名称             | 类型    | 必选项 | 默认值        | 有效值  | 描述                                             |
 | ---------------- | ------- | ------ | ------------- | ------- | ------------------------------------------------ |
-| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md)或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。请注意，**该设置是全局生效的**，因此在指定 log_format 后，将对所有绑定 clickhouse-logger 的 Route 或 Service 生效。 |
-
-### 设置日志格式示例
+| log_format       | object  | 否   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX](../apisix-variable.md) 或 [NGINX](http://nginx.org/en/docs/varindex.html) 变量。该配置全局生效。如果你指定了 `log_format`，该配置就会对所有绑定 `skywalking-logger` 的路由或服务生效。|
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/clickhouse-logger.md:56]:
```suggestion
| log_format       | object  | 否   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX](../apisix-variable.md) 或 [NGINX](http://nginx.org/en/docs/varindex.html) 变量。该配置全局生效。如果你指定了 `log_format`，该配置就会对所有绑定 `clickhouse-logger` 的路由或服务生效。|
```

**Review by SylviaBABY — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**